v2/support/modal_sample_app/README.md

```diff
@@ -0,0 +1,8 @@
+:warning: *Use of this software is subject to important terms and conditions as set forth in the License file* :warning:
+
+# Modal Sample App
+
+This sample app displays text on the ticket sidebar with an edit button. The edit button opens a modal where you can edit the text displayed. The text is saved on local storage.
+
+### Screenshot(s):
+![modal](https://cloud.githubusercontent.com/assets/2517811/19846439/5fa95d60-9f93-11e6-9559-3e2155e3990b.gif)
```

Comment:
will this url be long lived? do we normally do this or commit the gifs to the repo?